src/applications/claims-status/components/claim-files-tab/FilesOptional.jsx

```diff
@@ -1,28 +1,54 @@
 import React from 'react';
 import PropTypes from 'prop-types';
 import { Link } from 'react-router-dom-v5-compat';
+import { useFeatureToggle } from '~/platform/utilities/feature-toggles';
 
-import { truncateDescription } from '../../utils/helpers';
+import { truncateDescription, buildDateFormatter } from '../../utils/helpers';
 
 function FilesOptional({ item }) {
+  const { TOGGLE_NAMES, useToggleValue } = useFeatureToggle();
+
+  const cstFriendlyEvidenceRequests = useToggleValue(
+    TOGGLE_NAMES.cstFriendlyEvidenceRequests,
+  );
+  const dateFormatter = buildDateFormatter();
   return (
     <va-alert class="optional-alert vads-u-margin-bottom--2" status="info">
       <h4 slot="headline" className="alert-title">
-        {item.displayName}
+        {cstFriendlyEvidenceRequests && item.friendlyName
+          ? item.friendlyName
+          : item.displayName}
       </h4>
+      <p>Requested to others on {dateFormatter(item.requestedDate)}</p>
       <p className="alert-description">
-        {truncateDescription(item.description)}
+        {cstFriendlyEvidenceRequests &&
+        (item.shortDescription || item.activityDescription)
+          ? item.shortDescription || item.activityDescription
+          : truncateDescription(item.description)}
       </p>
-      <div className="call-to-action vads-u-padding-top--2">
-        You don’t have to do anything, but if you have this information you can{' '}
-        <Link
-          aria-label={`Add information for ${item.displayName}`}
-          className="add-your-claims-link"
-          to={`../document-request/${item.id}`}
-        >
-          add it here.
-        </Link>
-      </div>
+      {cstFriendlyEvidenceRequests ? (
+        <div className="call-to-action">
+          <Link
+            aria-label={`Add information for ${item.displayName}`}
+            className="add-your-claims-link"
+            to={`../document-request/${item.id}`}
+          >
+            About this notice
```

Comment:
This accessible label need to include the visible text "About this notice", or change the visible text to "Add information". Right now the visible label is ignored by screen readers for the `aria-label` which was a bit confusing and Voice Access can't click the link at all because of the difference.